code/modules/asset_cache/assets/cell_line.dm

```diff
@@ -0,0 +1,17 @@
+/datum/asset/spritesheet/cell_line
+	name = "cell_line"
+
+/datum/asset/spritesheet/cell_line/create_spritesheets()
```

Comment:
Consider using the new `dmicon` tgui component rather than a spritesheet

It's a lot easier / less painless to use (all you need to do is pass `icon` and `icon_state` in `ui_data`), faster for init, and looks better (IMO)